trade-finance-manager-ui/templates/case/tasks/tasks.njk

```diff
@@ -7,6 +7,10 @@
 
 {% extends "case/case.njk" %}
 
+{% block pageTitle -%}
+  Deal {{ deal.details.ukefDealId }} - Tasks
```

Comment:
GEF deals?

Reply:
Yes, it works for GEF, other deals, and all Facilities.